CHANGELOG.md

```diff
@@ -12,6 +12,128 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## 🛠 Maintenance
 ## 📚 Documentation -->
 
+# [0.0.5] - 2021-03-30
+
+> Important: X breaking changes below, indicated by **❗ BREAKING ❗**
+
+## 🚀 Features
+- **Adds introspection ability for subgraphs - [lrlna], [issue/349] [pull/377]**
+
+  A new command, `rover subgraph introspect` has been added. This command
+  runs a _federated introspection_ query against a server which has
+  implemented the requirements of the [federation
+  specification](https://www.apollographql.com/docs/federation/federation-spec/).
+  This command accepts endpoint [headers] for making the introspection
+  request (if required) and outputs SDL to stdout.
+
+  [lrlna]: https://github.com/lrlna
+  [pull/377]: https://github.com/apollographql/rover/pull/277
+  [issue/349]: https://github.com/apollographql/rover/issues/349
+
+- ** Fallback to monochromic output when `tput` is unavailable - [abernix], [issue/371] [pull/372]**
+
+  The `tput` command allows easier ANSI output using named values in rather than
+  control characters.
+
+  While we could just use control characters and maintain colored output in the
+  absence of `tput`, it's probably also reasonable to gracefully fall back to
+  monochromatic output.
+
+  [abernix]: https://github.com/abernix
+  [pull/372]: https://github.com/apollographql/rover/pull/372
+  [issue/371]: https://github.com/apollographql/rover/issues/371
+
+## ❗ BREAKING ❗
+- ** Renames `core build` to `supergraph compose` - [lrlna], [pull/391]**
+
+  To align with other Apollo teams on the usage `supergraph` and
+  `composition`, we are renaming `core build` to `supergraph compose`.
+
+  [lrlna]: https://github.com/lrlna
+  [pull/391]: https://github.com/apollographql/rover/pull/391
+
+- ** Updates harmonizer@0.2.2 - [abernix], [pull/396]**
+
+  Updates harmonizer to the latest version. This version now composes and
+  returns a core schema instead of CSDL. CSDL was an internal implementation
+  of composition and this new format is meant to bring stability to `rover
```

Comment:
```suggestion
  of composition and this new format is meant to bring some stability to `rover
```
idk if we need to hedge too much here but i don't necessarily wanna promise _too_ much stability